From bba2440b57e28d1a63e7bfe635f08908301b1500 Mon Sep 17 00:00:00 2001
From: OpenADK <wbx@openadk.org>
Date: Mon, 4 Jan 2021 16:44:11 +0100
Subject: [PATCH] Include a local copy of sys/queue.h instead of the libc one

Musl seems to not provide this header for whatever reason.
---
 support/include/conffile.h | 2 +-
 utils/mountd/v4root.c      | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/support/include/conffile.h b/support/include/conffile.h
index 7d974fe98f4f3..8d18cace2d5ee 100644
--- a/support/include/conffile.h
+++ b/support/include/conffile.h
@@ -33,7 +33,7 @@
 #ifndef _CONFFILE_H_
 #define _CONFFILE_H_
 
-#include <sys/queue.h>
+#include "queue.h"
 #include <ctype.h>
 #include <stdint.h>
 #include <stdbool.h>
diff --git a/utils/mountd/v4root.c b/utils/mountd/v4root.c
index dd9828eb0c114..4ef909f9e9a6c 100644
--- a/utils/mountd/v4root.c
+++ b/utils/mountd/v4root.c
@@ -13,7 +13,7 @@
 
 #include <sys/types.h>
 #include <sys/stat.h>
-#include <sys/queue.h>
+#include "queue.h"
 #include <stdio.h>
 #include <stdlib.h>
 #include <ctype.h>
